community/cypher/cypher/pom.xml

```diff
@@ -47,7 +47,7 @@
     <scala.binary.version>2.11</scala.binary.version>
     <maven.javadoc.failOnError>false</maven.javadoc.failOnError>
     <doclint-groups>none</doclint-groups>
-    <jetty.version>9.2.4.v20141103</jetty.version>
+    <jetty.version>9.3.5.v20151012</jetty.version>
```

Comment:
we have another jetty.version property defined at https://github.com/neo4j/neo4j/blob/3.0/community/server/pom.xml#L75
Why do you guys need another one?
If you can share the same version can you please move that property into common parent pom and reuse it?